src/vs/workbench/contrib/scm/browser/scmViewPane.ts

```diff
@@ -1789,7 +1789,8 @@ class SCMInputWidget {
 
 		const uri = URI.from({
 			scheme: Schemas.vscode,
-			path: `scm/${input.repository.provider.contextValue}/${input.repository.provider.id}/input`,
+			authority: 'scm',
+			path: `/${input.repository.provider.contextValue}/${input.repository.provider.id}/input`,
```

Comment:
Looks like GHPRI does rely on the old path, but it can be updated to look at the authority instead.

https://github.com/microsoft/vscode-pull-request-github/blob/f2ea88b0d3fab2e6c2c8a6952b628abca5eeaa83/src/issues/issueCompletionProvider.ts#L185-L186

Reply:
I have the change out for review. Feel free to merge already.